fix(lint/complexity/noUselessFragments): fix Fragments containing HTML entities marked as unnecessary (#6518)
biomejs/biome: .changeset/proud-windows-tease.md

---
"@biomejs/biome": patch
---

> Fixed #6508, where Fragments containing HTML entities were incorrectly marked as unnecessary.

Reviewer: ```suggestion
Fixed #6508, where the rule `noUselessFragments` incorrectly flagged Fragments containing HTML entities as unnecessary.
```
